operations/runbooks.rst

```diff
@@ -89,6 +89,4 @@ Zápis z jednorázového elektronického hlasování
 
       .. image:: ../_static/voting/YYYY-MM-DDTHH-MM-SS.png
 
-      .. image:: ../_static/voting/YYYY-MM-DDTHH-MM-SS.png
```

Comment:
This was supposed to be an example of when multiple screenshots are needed, but I agree it looks like a typo 😀

Reply:
I think that:
* if you just need one screenshot, this definitely looks like a typo
* if you need several screenshots, you can figure that out even without the duplicated line

Reply:
But I can revert if you want :)

Reply:
No, I approved it. No strong opinions, I agree with the change. It was more of an explanation of my original intentions for context.